pkg/server/container_create_windows.go

```diff
@@ -277,3 +298,65 @@ func (c *criService) generateContainerSpec(id string, sandboxID string, sandboxP
 
 	return g.Config, nil
 }
+
+// addOCIBindMounts adds bind mounts.
+func (c *criService) addOCIBindMounts(g *generate.Generator, mounts []*runtime.Mount) error {
```

Comment:
I think we need to be a little more careful in this part for Windows/Linux mounts. If the runtime is Windows we want to handle certain types of mounts and if the runtime is LCOW we want to allow things like "/dev". Does that make sense?

Reply:
But this is really close!

Reply:
Maybe we should just add a few tests actually and that would give us all the confidence we need. Basically for runtime.Mount types with "File, Folder, Disk, vhd" what OCI mount did it add for LCOW/WCOW and we would be golden.